### PR TITLE
Created MapCitiesDataServlet.java file

### PR DESCRIPTION
**Pull request by breelynbetts**
Added a csv file to WEB-INF folder and created a servlet that created json arrays out of the data

### CODE REVIEW

File: src/main/java/com/google/codeu/servlets/MapCitiesDataServlet.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
+package com.google.codeu.servlets;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonArray;
+
+import javax.servlet.annotation.WebServlet;
```
Comment (albfelix31) [inline src/main/java/com/google/codeu/servlets/MapCitiesDataServlet.java:8]: maybe consider putting the imports on alphabetical order, just for styling and organization purposes

File: src/main/java/com/google/codeu/servlets/MapCitiesDataServlet.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+ * Returns City data as a JSON array
+ * example: [{"lat": 34.567, "lng": -123.45}]
+ */
+@WebServlet("/map-data")
+public class MapCitiesDataServlet extends HttpServlet {
+
+  private JsonArray popCityArray;
+
+  @Override
+  public void init() {
```
Comment (albfelix31) [inline src/main/java/com/google/codeu/servlets/MapCitiesDataServlet.java:29]: what does this method do?

Comment (breelynbetts) [reply src/main/java/com/google/codeu/servlets/MapCitiesDataServlet.java:29]: This method reads each line of data at "/WEB-INF/popularCities.csv" and converts it to JSON, which is then added to an ArrayList. It's basically used to get the latitude and longitude data for the markers on the map.

Comment (albfelix31) [reply src/main/java/com/google/codeu/servlets/MapCitiesDataServlet.java:29]: Ah alright, I understand it now! :)

**Review by albfelix31 — APPROVED** [review]
LGTM! Thanks for explaining the method!

**Pull request merged**